src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -44,7 +44,7 @@
     "workspaceContains:**/*.csx",
     "workspaceContains:**/*.cake"
   ],
-  "main": "./dist/extension",
+  "main": "../Microsoft.AspNetCore.Razor.VSCode.Extension/dist/extension.js",
```

Comment:
This is a weird hack, but specifying only `./dist/extension` or `./dist/extension.js` doesn't work. We actually have to go up a folder and then down the same folder for the `extension.js` file to be recognized. Taylor and I looked into this issue together and determined that this is likely a bug, but we both agreed that this workaround is sufficient for now. 😅